src/ModFiles/Portal 2/install_dlc/scripts/vscripts/multiplayermod/config.nut

```diff
@@ -18,23 +12,30 @@
 // Set true/false if you want to see the P2:MM debug info in the console
 Config_DevMode <- false
 
+// Set true/false if you want to use a VScript Debugger. This also sets "developer" to 1. Only works for Windows systems!
+// Highly recommended to use the "VScript Debugger" by LionDoge, a extension for Visual Studio Code. https://marketplace.visualstudio.com/items?itemName=LionDoge.vscript-debug
+// How debugging was implemented for P2MM has been done according to LionDoge's extension instructions. 
+// This means that Config_DevMode will be automatically set to true wether or not it's been set to false as developer is needed to be enabled.
+// On every map change you will need to reattach the debugger, that's just unfortunately how Valve programmed it.
+//! ONLY ENABLE THIS IF YOU KNOW WHAT YOU'RE DOING! Game will freeze and won't unfreeze until a debugger is attached!
+Config_VScriptDebug <- false
+
 // Set true/false if you want to see the visual debug lines in-game for ray tracing
 Config_VisualDebug <- false
 
 // Set true/false if you are a developer and wish to load your custom player model in-game
+// Currently does not work on dedicated servers
 Config_UseCustomDevModels <- false
 
 //-----------------------------------
 // Game variables
 //-----------------------------------
 // If you want to change the gamemode on the next map load, set Config_GameMode to...
-// 0 - Standard   : A stable experience
-// 1 - Speedrun   : For those who like to speedrun Portal 2     (No implementation yet)
-// 2 - Deathmatch : Fight to the death in a team battle         (No full implementation yet)
-// 3 - Futbol     : Portal 2 beta minigame restored             (No full implementation yet)
+// 0 - Standard   : Most stable experience
+// 1 - Speedrun   : For those who like to speedrun Portal 2     (No full implementation yet)
 Config_GameMode <- 0
 
-// Set true/false if you want to randomize every Turret's models and colors (for fun lol)
+// Set true/false if you want to randomize every Turret's models and colors (for fun lol) (BROKEN D:)
```

Comment:
💔

Reply:
I think I could probably make it work again, I'll add it to my to do list.

Reply:
Yes, we are getting close